code/data.py

```diff
@@ -0,0 +1,80 @@
+'''
+Provide processed data ready for consumption by tensorflow model
+Return data always as numpy array
```

Comment:
Ok I can see what you're doing here but I'm generally not reading your code from beginning to end. I'm jumping to whatever method I want and then trying to figure out how to use it.